arch/arm/soc/st_stm32/stm32f2/dts.fixup

```diff
@@ -0,0 +1,49 @@
+/* SoC level DTS fixup file */
+
+#define CONFIG_NUM_IRQ_PRIO_BITS	        ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define CONFIG_UART_STM32_PORT_1_BASE_ADDRESS	ST_STM32_USART_40011000_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_1_BAUD_RATE	ST_STM32_USART_40011000_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_1_IRQ_PRI	ST_STM32_USART_40011000_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_PORT_1_NAME		ST_STM32_USART_40011000_LABEL
+#define PORT_1_IRQ				ST_STM32_USART_40011000_IRQ_0
+
+#define CONFIG_UART_STM32_PORT_2_BASE_ADDRESS	ST_STM32_USART_40004400_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_2_BAUD_RATE	ST_STM32_USART_40004400_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_2_IRQ_PRI	ST_STM32_USART_40004400_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_PORT_2_NAME		ST_STM32_USART_40004400_LABEL
+#define PORT_2_IRQ				ST_STM32_USART_40004400_IRQ_0
+
+#define CONFIG_UART_STM32_PORT_3_BASE_ADDRESS	ST_STM32_USART_40004800_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_3_BAUD_RATE	ST_STM32_USART_40004800_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_3_IRQ_PRI	ST_STM32_USART_40004800_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_PORT_3_NAME		ST_STM32_USART_40004800_LABEL
+#define PORT_3_IRQ				ST_STM32_USART_40004800_IRQ_0
+
+#define CONFIG_UART_STM32_PORT_6_NAME		ST_STM32_USART_40011400_LABEL
+#define CONFIG_UART_STM32_PORT_6_BASE_ADDRESS	ST_STM32_USART_40011400_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_6_BAUD_RATE	ST_STM32_USART_40011400_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_6_IRQ_PRI	ST_STM32_USART_40011400_IRQ_0_PRIORITY
+#define PORT_6_IRQ				ST_STM32_USART_40011400_IRQ_0
+
+#define CONFIG_UART_STM32_PORT_4_NAME		ST_STM32_USART_40004C00_LABEL
+#define CONFIG_UART_STM32_PORT_4_BASE_ADDRESS	ST_STM32_USART_40004C00_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_4_BAUD_RATE	ST_STM32_USART_40004C00_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_4_IRQ_PRI	ST_STM32_USART_40004C00_IRQ_0_PRIORITY
+#define PORT_4_IRQ				ST_STM32_USART_40004C00_IRQ_0
+
```

Comment:
Switching the console to UART4 generates:
zephyr/include/generated/generated_dts_board.h:185:47: error: 'ST_STM32_USART_40004C00_BASE_ADDRESS' undeclared here (not in a function); did you mean 'ST_STM32_UART_40004C00_BASE_ADDRESS'?
 #define CONFIG_UART_STM32_PORT_4_BASE_ADDRESS ST_STM32_USART_40004C00_BASE_ADDRESS
 
Correcting USART to UART in the subsequent lines the error disapears:

#define CONFIG_UART_STM32_PORT_4_NAME		ST_STM32_UART_40004C00_LABEL
#define CONFIG_UART_STM32_PORT_4_BASE_ADDRESS	ST_STM32_UART_40004C00_BASE_ADDRESS
#define CONFIG_UART_STM32_PORT_4_BAUD_RATE	ST_STM32_UART_40004C00_CURRENT_SPEED
#define CONFIG_UART_STM32_PORT_4_IRQ_PRI	ST_STM32_UART_40004C00_IRQ_0_PRIORITY
#define PORT_4_IRQ				ST_STM32_UART_40004C00_IRQ_0

Reply:
Thank you, it's my mistake and I had fix it.